flow_control/flow.py

```diff
@@ -206,7 +206,7 @@ def get_location_string(self, locator, is_draft=False):
                 type_id=resource,
                 locator=locator)
         else:
-            course_url = course_url.split(course_prefix)[-1]
+            course_url = course_url.replace(course_prefix, '', 1)
```

Comment:
Now it will take only the first occurrence of the word `course`, that represents the prefix of the course url. All the next `course` words will be omitted.